docs/generators/lunr.fsx

```diff
@@ -42,8 +42,8 @@ let generate (ctx : SiteContents) (projectRoot: string) (page: string) =
 
           let gen =
               let ctn =
-                  sprintf "%s \n %s" generatorOutput.AssemblyGroup.Name (generatorOutput.AssemblyGroup.Namespaces |> Seq.map (fun n -> n.Name) |> String.concat " ")
-              {uri = (rootUrl + sprintf "/reference/%s/index.html" n.Label ); title = sprintf "%s - API Reference" n.Label; content = ctn }
+                  $"%s{generatorOutput.AssemblyGroup.Name} \n %s{(generatorOutput.AssemblyGroup.Namespaces |> Seq.map (fun n -> n.Name) |> String.concat " ")}"
```

Comment:
The compiler doesn't seem to like the nested quotes in the ```String.concat " "``` here? 

![image](https://github.com/user-attachments/assets/0a7dc861-afe5-4bcc-9289-5e6d9446766a)